website/cue/reference/components/sources/splunk_hec.cue

```diff
@@ -82,6 +83,59 @@ components: sources: splunk_hec: {
 				}
 			}
 		}
+		indexer_acknowledgements: {
```

Comment:
@001wwang did you mean to include this? Shouldn't this be nested under `configuration.acknowledgements`?

Reply:
Ah, shoot, I forgot to update the docs after the refactoring. Thanks for catching! I'll follow up with a fix.